### PR TITLE
v2.8.0

### PR DESCRIPTION
**Pull request by IAlibay**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,22 +24,23 @@ requirements:
...
   run:
     - python
     - scipy >=1.5.0
-    - {{ pin_compatible('numpy', lower_bound='1.22.3') }}
+    - numpy >=1.23.2
     - matplotlib-base >=1.5.1
     - seaborn    # [not ppc64le]
     - netcdf4 >=1.0
-    - biopython >=1.80
+    - biopython >=1.80  # [not py>312]
```
Comment (orbeckst) [inline recipe/meta.yaml:43]: Does biopython not work for 3.13 yet?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,22 +24,23 @@ requirements:
...
     - numpy {{ numpy }}
     - cython
     - packaging
-
+    - setuptools
   run:
     - python
     - scipy >=1.5.0
-    - {{ pin_compatible('numpy', lower_bound='1.22.3') }}
+    - numpy >=1.23.2
```
Comment (orbeckst) [inline recipe/meta.yaml:39]: nice — I read the npy 2.0 migrator notes

File: recipe/meta.yaml [hunk 3]
```diff
@@ -66,8 +69,8 @@ test:
 
 about:
   home: https://www.mdanalysis.org
-  license: GPL-3.0-or-later
-  license_family: GPL
+  license: LGPL-3.0-or-later
+  license_family: LGPL
```
Comment (orbeckst) [inline recipe/meta.yaml:73]: big step forward!

**Review by orbeckst — APPROVED** [review]
- [x] LICENSE was properly updated.
- [x] numpy 2.0 migration looks good

LGTM, thanks for the update.

**Comment by orbeckst** [comment]
Obviously, we shouldn't merge until _all_ runners build correctly.

File: recipe/meta.yaml (hunk 1 above)
Comment (IAlibay) [reply recipe/meta.yaml:43]: Not as far as I can tell, their migrator is currently up.

**Comment by IAlibay** [comment]
Thanks @orbeckst - the main thing we need to do is make a new release of mdahole2, waterdynamics, and pathsimanalysis.

The former is most important because it has a py<3.13 pin on.

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@orbeckst we have a weird cycling dependency going on - I'm going to not add python 3.13 support for now and do it as a separate build.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -56,6 +56,9 @@ requirements:
     - pytng >=0.2.3    # [not ppc64le]
     - pyedr >=0.7.1
     - mda-xdrlib
+    - waterdynamics
```
[the remark this answers is not in the document]
Comment (orbeckst) [reply recipe/meta.yaml:59]: Waterdynamics is now LGPL https://github.com/MDAnalysis/waterdynamics/pull/41

**Comment by orbeckst** [comment]
> we have a weird cycling dependency going on - I'm going to not add python 3.13 support for now and do it as a separate build.

Ok – I wouldn't know how to. 

Is the issue that the kits depend on MDA and MDA depends on the kits?

File: recipe/meta.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (IAlibay) [reply recipe/meta.yaml:59]: Thanks, just waiting on the waterdynamics feedstock to update and we should be good.

**Comment by orbeckst** [comment]
Then perhaps we drop the dependency on kits in MDAnalysis and tell users in an error message that they need to install the kit? I know that this is not how SemVer works but it would fix the problem and be in the spirit of "deprecations".

**Comment by orbeckst** [comment]
Basically, the installation instructions would be
```
mamba install mdanalysis waterdynamics mdahole2 pathsimanalysis ## did not look up package names
```
instead of
```
mamba install mdanalysis
```

**Comment by IAlibay** [comment]
@orbeckst the issue is that the mdakits themselves need MDAnalysis... so it's just weird packaging wise.

That being said - I _think_ it works for now, at least the checks are passing. The issue was that py3.13 didn't work because it was attempting to first look for a compatible MDA version with py3.13 (for that mdakit) and then failing to resolve. Once 2.8.0, we can create a new build that should work.

**Comment by orbeckst** [comment]
You're in charge here — merge when ready. 🚀

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
Having to drop windows too...

**Comment by wshanks** [comment]
If you don't want to drop Windows, consider pinning conda-build until the issue is fixed:

https://github.com/conda/conda-build/issues/5416#issuecomment-2501464295

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@wshanks you are my hero 🚀

**Comment by orbeckst** [comment]
Thank you @IAlibay !!

**Comment by IAlibay** [comment]
Py13 support will have to be done elsewhere.. that might be a lot harder.

**Pull request merged**